docs/Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
-gem 'just-the-docs'
\ No newline at end of file
+gem 'just-the-docs'
+gem "webrick", "~> 1.8"
```

Comment:
just checking - this is new?

Reply:
Yes, without it I go this error: 

```
bundler: failed to load command: jekyll (/Users/rmoff/.gem/ruby/3.1.3/bin/jekyll)
LoadError: cannot load such file -- webrick
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve/servlet.rb:3:in `require'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve/servlet.rb:3:in `<top (required)>'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:184:in `require_relative'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:184:in `setup'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:102:in `process'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:93:in `block in start'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:93:in `each'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:93:in `start'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/lib/jekyll/commands/serve.rb:75:in `block (2 levels) in init_with_program'
  /Users/rmoff/.gem/ruby/3.1.3/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `block in execute'
  /Users/rmoff/.gem/ruby/3.1.3/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
  /Users/rmoff/.gem/ruby/3.1.3/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
  /Users/rmoff/.gem/ruby/3.1.3/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
  /Users/rmoff/.gem/ruby/3.1.3/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
  /Users/rmoff/.gem/ruby/3.1.3/gems/jekyll-3.9.3/exe/jekyll:15:in `<top (required)>'
  /Users/rmoff/.gem/ruby/3.1.3/bin/jekyll:25:in `load'
  /Users/rmoff/.gem/ruby/3.1.3/bin/jekyll:25:in `<top (required)>'
```

Reply:
> The jeklly version should match the one we use in the github workflows for the docs. The default is the one that github provides part of the base vm image. I suggest to match to the same version or have the actions install the version you like to use so we will have the same results.

What about if we specify it in the `Gemfile`, would that be the correct approach? That way local builds would match the action since IIUC the action [includes a bundle install](https://github.com/treeverse/lakeFS/blob/b8a7c496ebbe16762139ddece4008b6310e2891a/.github/workflows/docs-release.yaml#L24)